pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml elements
+
+> Extract elements and display structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements from an XML document (producing XPATH expressions):
+
+`xml elements {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements and attributes from an XML document:
+
+`xml elements -a {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Extract elements, attributes, and their values from an XML doc:
+
+`xml elements -v {{path/to/input.xml|URI}} >{{path/to/elements}}`
+
+- Print sorted unique elements from an XML document (to see structure):
```

Comment:
No guarantee, but I think it should be either `sorted, unique elements` or `sorted and unique elements`.
Also, I think there's no need to put that into parenteses:
```suggestion
- Print sorted unique elements from an XML document to see its structure:
```

Reply:
Agree. I added the parenthesis to the manual's description but I like yours better.

Reply:
xml-elements.md:- Print sorted unique elements from an XML document to see its structure:
xml-elements.md:- Print sorted unique elements from an XML document up to a depth of 3: